### PR TITLE
Feature/sinf 86 bat aurora

### PR DESCRIPTION
**Pull request by adrianmilne**
Will create an Aurora version of the Spree DB, provisioned in another Terraform project. I've run this on SBX1 and updated the service to use this new database (there will be a follow up PR to update the service TF - for now they can exist side-by-side).

I've also moved Elasticsearch into here - not sure if this is the best approach. I know you mentioned previously the possibility of creating another repo for that - which is another option happy to consider/implement (I think ES indexes the DB so they seemed quite connected).

I have a snapshot with prepopulated user data which the DB can be built from if needed (I can provide details)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+### Prerequisites
+
+Ensure that the following SSM parameters have been configured in AWS:
+
+```
+/bat/{env}-spree-db-master-username
+/bat/{env}-spree-db-master-password
+
+/bat/{env}-spree-db-app-username
+/bat/{env}-spree-db-app-username
```
Comment (tommyb82) [inline README.md]: `/bat/{env}-spree-db-app-password`

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+### Post Create
+
+After the database has been provisioned, we need to create a user for the spree application to use to connect.
+
+```
+CREATE USER {USERNAME} WITH ENCRYPTED PASSWORD '{PASSWORD}';
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO {USERNAME};
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO {USERNAME};
+GRANT ALL PRIVILEGES ON DATABASE spree TO {USERNAME};
```
Comment (tommyb82) [inline README.md:29]: Was wondering if we could do this with a local-exec provisioner - probably, but would involve installing and configuring psql client tooling as part of the CodeBuild install phase - and picking out the SSM params to use in the command - not worth it now but something we might want to look at for future DB provisioning and/or running our version update scripts as part of a DB pipeline??

**Review by tommyb82 — APPROVED** [review]
Awesome, up and running on SBX2 just doing the bat infra services 👍

File: README.md (hunk 1 above)
Comment (adrianmilne) [reply README.md]: fixed 👍

File: README.md (hunk 2 above)
Comment (adrianmilne) [reply README.md:29]: Yep - be good to automate the db scripts (certainly the version update scripts for prod releases) at some point if we get the time.

This wasn't the most elegant solution, but pragmatic for now given the time constraints. Maybe if it turns out to be troublesome we can revisit it.

**Pull request merged**